backend/src/engine/history-query-engine.ts

```diff
@@ -69,28 +66,18 @@ export default class HistoryQueryEngine extends BaseEngine {
       this.historyQueries.delete(settings.id);
       this.historyQueries.set(settings.id, historyQuery);
     }
-    this.historyQueries
-      .get(settings.id)!
-      .start()
-      .catch(error => {
-        this.logger.error(error);
-      });
+    await this.startHistoryQuery(settings);
```

Comment:
`startHistoryQuery` will actually be called twice, if the user creates a new history query:
1. First of all, the user calls the endpoint to create the query, which calls the reload service:
https://github.com/OptimistikSAS/OIBus/blob/650c83a3f23ccea1ca7d363525d373d6dc6a2a42/backend/src/web-server/controllers/history-query.controller.ts#L128
2. The reload service then creates and starts the HQ as well:
https://github.com/OptimistikSAS/OIBus/blob/650c83a3f23ccea1ca7d363525d373d6dc6a2a42/backend/src/service/reload.service.ts#L318-L319
3. But because `createHistoryQuery` calls `startHistoryQuery`:
_see code refferenced in this comment_
4. The `startHistoryQuery` gets called a second time, in the reload service:
_see point 2._

This can be fixed in two ways, either by removing the `startHistoryQuery` from `createHistoryQuery`, or from the reload service. I think removing it from the reload service might be the good idea, but I'm not 100% sure.